lib/zendesk_apps_support/app_version.rb

```diff
@@ -31,7 +35,15 @@ def valid_for_update?
     end
 
     def deprecated?
-      @version == DEPRECATED
+      DEPRECATED.include?(@version)
+    end
+
+    def sunsetting?
+      SUNSETTING && SUNSETTING.include?(@version)
```

Comment:
At the moment, yes, but this is preparing for when there are no versions currently being sunset which will happen sometime in the future.